src/formatstringplus/widget/formatstringplus.js

```diff
@@ -1,7 +1,16 @@
-
-dojo.provide("formatstringplus.widget.formatstringplus");
-
-dojo.declare('formatstringplus.widget.formatstringplus', mxui.widget._WidgetBase, {
+define([
+    "dojo/_base/declare",
+    "mxui/widget/_WidgetBase",
+    "mxui/dom",
+    "dojo/_base/lang",
+    "dojo/dom-construct",
+    "dojo/dom-class",
+    "dojo/date/locale"
+
+], function (declare, _WidgetBase, dom, lang, construct, domClass, locale) {
+    "use strict";
+
+    return declare("formatstringplus.widget.formatstringplus", [ _WidgetBase ], {
     
     _hasStarted         : false,
     _mxobj              : null,

```

Comment:
`start`  life cycle method is probably not right. Should be com `postCreate` ?

Reply:
Fixed